src/main.cpp

```diff
@@ -202,6 +206,7 @@ class BroadcastMessageDispatcher : public AbstractTask<3 * configMINIMAL_STACK_S
             if (!dispatcher.deserializeAndDispatch()) {
                 m_logger.log(LogLevel::Error, "Fail to deserialize/dispatch broadcast");
             }
+            Task::delay(50);
```

Comment:
Alot of delay were added, jsute to be sure, are they supposed to still be here or it was for debugging?

Reply:
I added them for safety, to make sure the queues are not locked by reading them with a mutex